### PR TITLE
[Process] - update @throws phpdoc

### PR DESCRIPTION
**Pull request by ybenhssaien**
| Q             | A
| ------------- | ---
| Branch?       | 4.4
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no
| Tickets       | no
| License       | MIT
| Doc PR        | 

Update PHPDoc throws Exceptions on https://github.com/symfony/symfony/blob/4.4/src/Symfony/Component/Process/Process.php

### CODE REVIEW

File: src/Symfony/Component/Process/Process.php [hunk 1]
```diff
@@ -393,9 +393,9 @@ public function restart(callable $callback = null, array $env = []): self
      *
      * @return int The exitcode of the process
      *
-     * @throws RuntimeException When process timed out
-     * @throws RuntimeException When process stopped after receiving signal
-     * @throws LogicException   When process is not yet started
+     * @throws ProcessTimedOutException When process timed out
+     * @throws ProcessTimedOutException When process stopped after receiving signal
```
Comment (nicolas-grekas) [inline src/Symfony/Component/Process/Process.php]: ProcessSignaledException?

File: src/Symfony/Component/Process/Process.php [hunk 2]
```diff
@@ -393,9 +393,9 @@ public function restart(callable $callback = null, array $env = []): self
...
+     * @throws LogicException           When process is not yet started
```
Comment (nicolas-grekas) [inline src/Symfony/Component/Process/Process.php]: \LogicException

Comment (nicolas-grekas) [reply src/Symfony/Component/Process/Process.php]: but actually the `\` should be removed from the code, and this one kept as is

**Comment by nicolas-grekas** [comment]
(for 4.3)

File: src/Symfony/Component/Process/Process.php (hunk 1 above)
Comment (youssefbenhssaien) [reply src/Symfony/Component/Process/Process.php]: Changed to `ProcessTimedOutException`

**Comment by nicolas-grekas** [comment]
Thank you @youssefbenhssaien.

**Pull request merged**